sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/SparkSqlLexer.g

```diff
@@ -328,6 +328,8 @@ KW_WEEK: 'WEEK'|'WEEKS';
 KW_MILLISECOND: 'MILLISECOND'|'MILLISECONDS';
 KW_MICROSECOND: 'MICROSECOND'|'MICROSECONDS';
 
+KW_NATURAL: 'NATURAL';
```

Comment:
Is the ```NATURAL``` keyword reserved? If it is not, please add it to ```nonReserved``` rule in the Expression parser.